ssn/rdf/sosa.ttl

```diff
@@ -281,6 +324,11 @@ sosa:isResultOf
   owl:inverseOf sosa:hasResult ;
   skos:definition "Relation linking a Result to the Observation or Actuation that created or caused it."@en ;
 .
+sosa:isSamplingResultOf
+  rdf:type owl:ObjectProperty ;
+  rdfs:label "is result of Sampling (activity)"@en ;
```

Comment:
I would suggest to remove (activity). Also, this seems to confuse/mix the label with a (missing) definition. The label should just be 'is sampling result of', right?